spec/factories/topics.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id          :bigint           not null, primary key
+#  description :string(50)       not null
+#  image       :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+FactoryBot.define do
+  factory :topic do
+    trait :music do
```

Comment:
maybe, is a good idea add a gem named faker https://github.com/faker-ruby/faker

This gem will help you to add random data in your factories